dask_expr/tests/test_shuffle.py

```diff
@@ -640,7 +641,7 @@ def test_set_index_sort_values_one_partition(pdf):
 
 def test_set_index_triggers_calc_when_accessing_divisions(pdf, df):
     divisions_lru.data = OrderedDict()
-    query = df.set_index("x")
+    query = df.fillna(random.randint(1, 100)).set_index("x")
```

Comment:
This one is funny, the previous expression was cached, so we were never calculating divisions with this change. Using a random number here avoid this